styles/style.css

```diff
@@ -0,0 +1,358 @@
+footer {
+  background-color: #037EAA;
+  padding: 2.5rem;
+}
+footer ul {
+
+  text-align: center;
+  padding-top: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+footer img {
+  max-width: 30%;
+  display: block;
+  padding-bottom: 1.2rem;
+}
+footer ul li {
+  display: inline-flex;
+}
+footer ul li a {
+  color: white;
+}
+footer ul li:not(:last-child)::after {
+  content: '/';
+  color: white;
+  padding: 0.5rem;
+}
+footer p {
+  color: white;
+  text-align: center;
+}
+/* footer style end */
+/* dog-card style start */
+.container-dog-card {
+  border: 2px solid #EBEBEB;
+  padding-bottom: 0.7rem;
+}
+.button-dog-card {
+  background-color: #81B741;
+  color: white;
+  font-weight: bold;
+  width: 90%;
+  border-radius: 2px;
+  border: none;
+  padding: 0.4rem 3rem 0.4rem 3rem;
+  display: flex;
+}
+.name-dog-card {
+  padding: 0.3rem 5% 0.3rem 5%;
+  color: #103E67;
+  font-size: 0.95rem;
+}
+.description-dog-card {
+  padding: 0.3rem 5% 0.6rem 5%;
+}
+.label-cost-dog-card, .description-dog-card, .number-cost-dog-card {
+  font-size: 0.7rem;
+  color: black;
+  font-family: sans-serif;
+}
+.label-cost-dog-card {
+  font-weight: bold;
+  display: inline;
+  padding: 0.3rem 0 0.3rem 5%;
+}
+.number-cost-dog-card{
+  display: inline;
+}
+/* dog-card style end */
+/* home page style start */
+.home-page {
+  display: flex;
+  flex-direction: column;
+}
+.home-page div:first-child {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  padding: 1.5rem 0 1.5rem 0;
+}
+.home-page >div:first-child h2 {
+  color: #103E67;
+  padding-bottom: 0.8rem;
+}
+
+.home-page div:first-child img {
+  padding-right: 2.5rem;
+}
+.home-page >div:nth-child(2) {
+  padding: 0 0 2rem 0;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 2.5rem 2.5rem;
+}
+.home-page button:nth-child(3) {
+  padding: 0.6rem 2.5rem 0.6rem 2.5rem;
+  color: #103E67;
+  border: none;
+  border-radius: 2px;
+  font-size: 0.8rem;
+  font-weight: bold;
+}
+/* home page style end */
+/* dogs page style start */
+.dogs-card-container4all {
+  padding:0;
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 2.5rem 2.5rem;
+}
+.dogs-page h1 {
+  color: #103E67;
+  text-align: left;
+  padding: 1.5rem 0 1.6rem 0;
+  font-size: 1.7rem;
+}
+/* dogs page style end */
+
+/* blog page style start */
+.blog-page {
+  padding: 0.5rem 9.5rem 5rem 9.5rem;
+  display: grid;
+  grid-template-rows: 10% 1fr 1fr 1fr;
+  grid-template-columns: 33% 67%;
+  grid-row-gap: 1rem;
+}
+.blog-page img {
+  padding: 1rem 2rem 1rem 0;
+}
+.blog-page h1 {
+  grid-area: 1 / 1 / 2 / 2;    /* a good way to align it left*/
+  color:  #103E67;
+  font-size: 1.5rem;
+  margin: 0;
+  padding: 1.3rem 0 0 0;
+}
+.blog-page h2 {
+  color:  #103E67;
+  font-size: 1.2rem;
+  margin: 0;
+  padding: 1rem 0;
+}
+.blog-page p {
+  color: black;
+  font-size: 0.83rem;
+  padding-bottom: 1rem;
+}
+.blog-page img:nth-child(2) {
```

Comment:
this strategy isn't the best. You'll want to wrap each blog in a container (like the dog tiles) and then lay them out. Plus you won't have to do all this nth-child business.